styles/MoodEntry.module.css

```diff
@@ -28,15 +28,12 @@
   border-radius: 3px;
 }
 
-.submit {
+.submitAnchor {
+  background-color: grey;
   width: 100%;
-  text-align: center;
+  height: 100%;
   padding: 10px;
-  background-color: grey;
   margin-top: 10px;
-  border-radius: 4px;
-}
-
-.submitAnchor {
-  color: white;
+  border: black 2px solid;
+  border-radius: 3px;
 }
```

Comment:
This is mostly picking nits and can be completely handled by your editor, but is good to know anyway. Unless you specifically know otherwise, it's safe to assume that any code you write will be served from a *nix system, and on *nix systems, by definition, text files are lines of text where each line is terminated with a newline character (`\n`). Github is screaming at you here because the last line of the file doesn't have a newline character terminating it. 

This is something you can set your IDE to do by default. Google "vs code trailing newline" or something like that to find out how to do it.

Here's a really good explanation of why this (seemingly pedantic nuisance) can potentially be meaningful. https://unix.stackexchange.com/a/18789